toplevel/ccompile.ml

```diff
@@ -147,19 +149,22 @@ let compile opts stm_options injections copts ~echo ~f_in ~f_out =
       let _doc = Stm.join ~doc:state.doc in
       let wall_clock2 = Unix.gettimeofday () in
       check_pending_proofs long_f_dot_in;
-      (* In .vo production, dump a complete .vo file.
-         In .vok production, only dump an empty .vok file. *)
+      (* In .vo production, dump a complete .vo file. *)
       if mode = BuildVo
         then Library.save_library_to ~output_native_objects Library.ProofsTodoNone ldir long_f_dot_out;
```

Comment:
I was surprised this could be blamed on 797eb524853e361f84c9c776024c142107f0c282, because that's old and Coq used to work fine — but the commit that you linked had `else create_empty_file long_f_dot_out;` here, which would end up creating a `vok` file.

That line was instead removed in https://github.com/coq/coq/commit/70fecff5d3ddbf4ebd5674cf2355ed7dbabf948a#diff-586a931ea3c914a1f3d5010067eeec906e413c2995c33cbbf2968910f9b19414 — which was merged between 8.14 and 8.15!

Reply:
Well spotted! Will update commit message.